### PR TITLE
patch `prune_dq_for_future_installed` out

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Fixes https://github.com/conda/conda-libmamba-solver/issues/317. See there for a beautiful journey down the rabbit hole.

TLDR: https://github.com/openSUSE/libsolv/commit/4edf4e62db655734d0c362c59f8cb1987ef6b164 seems to introduce non-deterministic behaviour that manifests in some tests in the conda test suite with alternative solutions. This reverts part of that PR so the new functions are not called in the main method.

### CODE REVIEW

**Comment by jaimergp** [comment]
On hold. I might be seeing the same issue in libsolv 0.7.24 so maybe this was a red herring.
